code/datums/gamemode/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -633,3 +633,54 @@
 	my_fac.HandleRecruitedRole(new_role)
 	new_role.Greet(GREET_DEFAULT)
 	new_role.AnnounceObjectives()
+
+
+
+//////////////////////////////////////////////
+//                                          //
+//             XENOMORPHS                   ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/midround/from_ghosts/faction_based/xenomorphs
+	name = "Alien Infestation"
+	role_category = /datum/role/xenomorph
+	enemy_jobs = list("AI", "Cyborg", "Security Officer", "Warden","Detective","Head of Security", "Captain", "Roboticist")
```

Comment:
Roboticist?

Reply:
I thought that it would be fitting since they can and most likely will build combat mechs